nextest-runner/src/reporter/displayer.rs

```diff
@@ -1982,14 +1982,15 @@ fn write_final_warnings(
             if cancel_status == Some(CancelReason::TestFailure) {
                 writeln!(
                     writer,
-                    "{}: {}/{} {} {} not run due to {} (run with {} to run all tests)",
+                    "{}: {}/{} {} {} not run due to {} (run with {} to run all tests, or run with {})",
```

Comment:
Could you reword this to:

`specify --no-fail-fast to run all tests, or --max-fail`

?